diesel/src/macros/mod.rs

```diff
@@ -632,6 +632,15 @@ macro_rules! table_body {
         current_column_doc = [],
         current_column_sql_name = [],
     ) => {
+        $(static_cond! {
+            if $table_name == $column_name {
+                compile_error!(concat!(
+                    "Column ",
+                    stringify!($column_name),
+                    " cannot be named the same as its table"
```

Comment:
Maybe you should point to
 ```
#[sql_name = "users"]
users_
```

Reply:
Ah, good point! I totally forgot about this.